src/fsd/4-entities/equipment/equipment.service.ts

```
@@ -0,0 +1,121 @@
+
+export class EquipmentService {
+    static readonly equipmentData: IEquipment[] = this.convertEquipmentData();
+
+    /**
+     * Converts the raw equipment data from JSON into something that more
+     * strongly typed.
+     */
+    private static convertEquipmentData(): IEquipment[] {
+        const ret: IEquipment[] = [];
+        Object.entries(equipmentData).forEach(([_, equipment]) => {
+            const slot = this.parseEquipmentType(equipment.slot);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)